### DISCUSSION TITLE
Stars from orgs not included

### DISCUSSION THREAD

**Discussion by JKamsker** [anuraghazra/github-readme-stats#3335] [Q&A] [answered]
Hey, my badge(https://github-readme-stats.vercel.app/api?username=jkamsker) (33) does not include the stars of one of my most popular repos https://github.com/audible-tools/audible-tools.github.io. (272 stars)

Also, total commits is also way too low.

Am i doing something wrong?

**Comment by qwerty541** [comment, marked as answer]
Hey, @JKamsker! Currently this project does not support organizations stats, you can track progress on this feature here https://github.com/anuraghazra/github-readme-stats/issues/1.

About the low commits count, i see that you have some private contributions in past monthes, by default, the card shows statistics only from public repositories. To show private statistics on the stats card, you should [deploy your own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using your own GitHub API token.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/d2745ae3-8a5b-47f5-b776-9d0594c2e60f)

**Reply by JKamsker** [reply]
Oh, i didn't know that, thanks for the clarification!
